docker/data-science/notebooks/ch05/3.semantic-knowledge-graph.ipynb

```diff
@@ -332,47 +330,47 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Simple Query Expansion:\n",
-      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'params': {'qf': 'title body', 'defType': 'edismax', 'mm': '0%'}}\n",
+      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'limit': 10, 'params': {'defType': 'edismax', 'ident': 'true', 'omitHeader': 'true', 'qf': ['title', 'body'], 'mm': '0%'}}\n",
```

Comment:
If we're going with Solr syntax then the default params shouldn't be printing out here... (omitHeader, limit, indent).

HOWEVER I actually don't think we need Solr syntax here at all. I think we should just use our book syntax (i.e. `query_fields` instead of `params: {`qf` ..}`.

I don't think these queries need to be Solr-specific, do they?

Reply:
I think they're more valuable as solr queries, or if the engine is changed it will print out that syntax. I'll move indent and omitheader to the schema, it was in the request transformer

Reply:
ok, got it. Go for it, and I'll take a look at this again when I review the ch5 manuscript changes.